content/docs/state-and-lifecycle.md

```diff
@@ -1,23 +1,24 @@
 ---
 id: state-and-lifecycle
-title: State and Lifecycle
+title: राज्य और जीवनचक्र
 permalink: docs/state-and-lifecycle.html
 redirect_from:
   - "docs/interactivity-and-dynamic-uis.html"
 prev: components-and-props.html
 next: handling-events.html
 ---
 
-This page introduces the concept of state and lifecycle in a React component. You can find a [detailed component API reference here](/docs/react-component.html).
+यह पृष्ठ एक प्रतिक्रिया घटक में स्थिति और जीवनचक्र की अवधारणा का परिचय देता है। आप [यहाँ एक विस्तृत घटक API संदर्भ पा सकते हैं](/docs/react-component.html)।
```

Comment:
`आप [यहाँ एक विस्तृत घटक API संदर्भ पा सकते हैं](/docs/react-component.html)।`
=>
`आप [यहाँ एक विस्तृत कौम्पोनॅन्ट API संदर्भ](/docs/react-component.html) पा सकते हैं।`

Reply:
There is one more issue in this

`html)पा सकते हैं।` => `html) पा सकते हैं।`

you forgot to add a space before `पा`